src/script/components/resource-hub.ts

```diff
@@ -193,6 +235,10 @@ export class ResourceHub extends LitElement {
 
   constructor() {
     super();
+
+    window.addEventListener('resize', () => {
+      this.requestUpdate();
```

Comment:
Why - is this because we don't want to render blog actions on small screens?

Reply:
yes, at least for development. But if a user decides to resize the screen the resize event doesn't trigger with the webcomponent, hence the listener on the window. We could just hide elements in the dom, which would be simpler too.

Reply:
Yeah, since CSS classes (hidden-sm, etc.) would work here without the need for resize listeners, let's switch to that. Can you update this PR?